src/android/app/build.gradle.kts

```diff
@@ -32,12 +32,12 @@ android {
     ndkVersion = "27.1.12297006"
 
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_17
-        targetCompatibility = JavaVersion.VERSION_17
+        sourceCompatibility = JavaVersion.VERSION_21
+        targetCompatibility = JavaVersion.VERSION_21
```

Comment:
Is there any particular reason why the Java version was updated here?
I tried reverting the commit that introduced this, and it builds and runs as I would expect on Android 15.

Reply:
Sorry; that was a copy/paste holdover from my testing fork. JDK 17 should be good until 2029, although who knows when the Android SDK will start to take advantage of features in newer JDKs (FWIW, 21 is LTS too). But that can be dealt with when the time comes. I've reverted it.